py/TelemMessages/Waypoint.py

```diff
@@ -0,0 +1,67 @@
+"""LCM type definitions
+This file automatically generated by lcm.
+DO NOT MODIFY BY HAND!!!!
+"""
+
+try:
+    import cStringIO.StringIO as BytesIO
+except ImportError:
+    from io import BytesIO
+import struct
+
+class Waypoint(object):
+    __slots__ = ["longitude", "latitude", "altitude", "waypoint_id"]
```

Comment:
is this the order we decided on? there's lat lon earlier and now it's lon lat...

Reply:
https://uwarg-docs.atlassian.net/wiki/spaces/ARCHS22/pages/2131329241/Communication+Message+Formats
I think for waypoints its meant to be this order, for odometry its the other way around. Its consistent with the doc, but we can change the doc, since it seems that efs kind of owns all of the telem format stuff now

Reply:
let's decide on a common format (lat lon) https://en.wikipedia.org/wiki/ISO_6709